extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.css

```diff
@@ -40,4 +43,129 @@ amp-story-subscriptions.i-amphtml-story-subscriptions-visible::before {
 amp-subscriptions-dialog.i-amphtml-story-subscriptions-dialog-wrapper {
   border-radius: 16px 16px 0px 0px !important;
   font-family: Poppins, sans-serif !important;
+  font-weight: 400 !important;
+  background: white !important;
+  color: black !important;
+}
+
+.i-amphtml-story-subscriptions-dialog-banner {
+  position: relative !important;
+  width: 100% !important;
+  border-radius: 16px 16px 0px 0px !important;
+  font-size: 24px !important;
+  text-align: center !important;
```

Comment:
Is there ever text content here? If not then we should remove this and add it back when that functionality is added.

Reply:
Removed